Spec.md

```diff
@@ -0,0 +1,223 @@
+# OPC UA Pub/Sub Interface Description
+
+This document defines the Interface of the umati Dashboard based on OPC UA PubSub.
+It describes how a OPC UA server address space must be mapped to OPC UA PubSub, so that the infromation can map to the templates and displayed.
+
+## Requirements for the Mapping of the address space to PubSub
+
+(Reference: [umati/UA-CloudPublisher#5](https://github.com/umati/UA-CloudPublisher/issues/5))
+
+- Ability to react to dynamic changes in the address space (nodes added/deleted)
+  - Changes should be detected by monitoring NodeManagement events.
+- Resolve references to nodes (NodeId as values)
+- Hierarchy references must be mapped
+- Non-hierarchy references should be mapped if necessary
+  - Non-hierarchy references are mapped if relevant to the data semantics.
+- Provide type information
+- Methods should be considered
+- Online detection (possibly as separate topics)

```

Comment:
use the anyway mandatory `ua-status` message and topic:
https://reference.opcfoundation.org/Core/Part14/v105/docs/7.2.5.5.5
_IsCyclic_ shall be mandatory

Reply:
see: https://github.com/umati/UA-CloudPublisher/blob/c2e0a5083d1470f415dca6d03693e722d90a8e4c/Spec.md#L211-L212